pkg/openscap/openscap.go

```diff
@@ -109,11 +110,18 @@ func (s *defaultOSCAPScanner) getInputCVE(dist int) (string, error) {
 	return cveFileName, err
 }
 
+func strOrDefault(s string, d string) string {
+	if len(s) == 0 { // s || d
+		return d
+	}
+	return s
+}
+
 func (s *defaultOSCAPScanner) setOscapChrootEnv() error {
 	for k, v := range map[string]string{
 		"OSCAP_PROBE_ROOT":              s.imageMountPath,
 		"OSCAP_PROBE_OS_VERSION":        "4.2.6-301.fc23.x86_64", // FIXME place holder value
```

Comment:
@enoodle what's this place holder??? please fix with something more appropriate.